### PR TITLE
Connected user login to backend API

### PR DESCRIPTION
**Pull request by JayachandraGunturu**

### CODE REVIEW

**Review by Satya-Irrinki — APPROVED** [review]

**Review by Satya-Irrinki — COMMENTED** [review]
Please remove console logs.

**Review by Satya-Irrinki — CHANGES_REQUESTED** [review]
Please remove console logs

File: workspaces/client/src/LoginComponent/LoginComponent.tsx [hunk 1]
```diff
@@ -49,6 +50,12 @@ export default function SignIn() {
       validationErrors.isEmailError = true;
     }
     else{
+      const url = `http://localhost:4000/login`;
+      console.log(loginData.password);
```
Comment (venkata-sai-samanth-kommi) [inline workspaces/client/src/LoginComponent/LoginComponent.tsx]: remove console logs

File: workspaces/client/src/LoginComponent/LoginComponent.tsx [hunk 2]
```diff
@@ -49,6 +50,12 @@ export default function SignIn() {
...
+      axios.post(url,{...loginData})
```
Comment (venkata-sai-samanth-kommi) [inline workspaces/client/src/LoginComponent/LoginComponent.tsx:54]: do use this, instead use the existing redux syntax

File: workspaces/client/src/LoginComponent/LoginComponent.tsx [hunk 3]
```diff
@@ -49,6 +50,12 @@ export default function SignIn() {
...
+      .then((response)=>{
+        console.log(response.data.approved);
```
Comment (venkata-sai-samanth-kommi) [inline workspaces/client/src/LoginComponent/LoginComponent.tsx:56]: remove log

**Review by venkata-sai-samanth-kommi — CHANGES_REQUESTED** [review]

**Pull request merged**